crates/turborepo-telemetry/src/events.rs

```diff
@@ -0,0 +1,40 @@
+use serde::{Deserialize, Serialize};
+
+/// All possible telemetry events must be included in this enum
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
As a general rule of thumb, I find it has been helpful to write unit tests for all `Serialize` implementations just to confirm that the derived implementation is:
 - what you expect
 - doesn't change unexpectedly due to something like a field rename

Example: https://github.com/vercel/turbo/blob/main/crates/turborepo-api-client/src/spaces.rs#L254

Reply:
Nice, I added one for now but will add more once the event structure gets finalized.